bundles/AdminBundle/Resources/public/js/pimcore/object/folder/search.js

```diff
@@ -202,15 +202,21 @@ pimcore.object.search = Class.create(pimcore.object.helpers.gridTabAbstract, {
         // get current class
         var classStore = pimcore.globalmanager.get("object_types_store");
         var klass = classStore.getById(this.classId);
-        var baseParams = {
-            language: this.gridLanguage,
-        };
+
+        var baseParams;
```

Comment:
It should be changed for variants tab as well. https://github.com/pimcore/pimcore/blob/f9fc337d234a92d1f30dbad53739b66bb71020f5/bundles/AdminBundle/Resources/public/js/pimcore/object/variantsTab.js#L91